src/components/UserCard.tsx

```diff
@@ -4,31 +4,68 @@ import { UserCardType } from "../types/UserCard";
 
 const UserCard: React.FC<UserCardType> = (props: UserCardType) => {
   const { name, profile } = props;
-  const { description, school, year, skills, commitmentLevel } = profile;
+  const { description, school, year, major, skills, commitmentLevel } = profile;
+
+  // Custom style for the school section if it's "Georgia Institute of Technology"
+  const schoolStyle =
+    school === "Georgia Institute of Technology"
+      ? {
+          color: "gold",
+          fontWeight: "bold",
+        }
+      : {};
+
+  // Custom style for the commitment tag based on the commitment level
+  const commitmentStyle = 
+  commitmentLevel === "low"
+    ? {
+        bg: "red.400",
+        color: "white",
+      }
+    : commitmentLevel === "medium"
+    ? {
+        bg: "yellow.400",
+        color: "black",
+      }
+    : commitmentLevel === "high"
+    ? {
+        bg: "green.400",
+        color: "white",
+      }
+    : {};
+
```

Comment:
I like your initiative here, and yeah we should def have something unique for Georgia Tech and [color coding](https://github.com/HackGT/match/issues/7) for commitment level. I actually implemented a bee avatar for GT, you can check it out rn at [match.hexlabs.org](https://match.hexlabs.org/). Open to further suggestions, but can you remove these changes?